alpine/8u252-8.46.0.19-jre-headless/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM alpine:3.10
+
+ENV LANG en_US.UTF-8
+ENV LANGUAGE en_US:en
+ENV LC_ALL en_US.UTF-8
+
+RUN wget --quiet https://cdn.azul.com/public_keys/alpine-signing@azul.com-5d5dc44c.rsa.pub -P /etc/apk/keys/ && \
+    echo "https://repos.azul.com/zulu/alpine" >> /etc/apk/repositories && \
+    apk --no-cache add zulu8-jre-headless
```

Comment:
@ironstas1k @sgrinev In previous versions ([8u232-8.42.0.23](https://github.com/zulu-openjdk/zulu-openjdk/blob/master/alpine/8u232-8.42.0.23-jre-headless/Dockerfile), [8u242-8.44.0.11](https://github.com/zulu-openjdk/zulu-openjdk/blob/master/alpine/8u242-8.44.0.11-jre-headless/Dockerfile), package versions were pinned. Was this change intentional?